webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -400,6 +400,33 @@ def update_response_parameter_scatterplot(
     def update_corr_index(
         click_data: Dict, responses: List[str], corr_xindex: Dict, corr_param_resp: Dict
     ) -> Optional[Dict]:
+        """
+        One of the data points we keep track of in the response correlation
+        plugin is the current index, representing a point on the x-axis, for
+        each selected response.
+        Some data shown is a function of the x value represented by that index,
+        for example the correlation scatterplot, as well as the heatmap and the
+        tornado plot.
+
+        This function updates a dictionary with responses as keys, and the
+        current index for each response as values.
+
+        It gets triggered when a user
+        - clicks on the response overview plot, effectively choosing an index
+        - changes the selection of responses
```

Comment:
I would add: ", in this case a default value is stored".

Reply:
indeed, thank you!